science/ecCodes/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup cmake     1.1
 PortGroup compilers 1.0
 
 name                ecCodes
-version             2.17.0
+version             2.18.0
 revision            1
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
Sorry, forgot. Done.